package.json

```diff
@@ -20,7 +20,7 @@
   },
   "dependencies": {
     "ocaml": "4.12.",
-    "@opam/dune": "*",
+    "@opam/dune": "~2.9.3",
```

Comment:
What about opening an issue to track this trouble? (with a dedicated label like `dependencies` or even `technical debt`)
From my point of view, this is bad habit to lock such a dependency. But this is understandable since it's a major release (2.9.2 to 3.X.X).
What do you think?

Reply:
#451